### PR TITLE
LOG-5906: Change component selectors to use kubernetes recommended

### PR DESCRIPTION
**Pull request by jcantrill**
labels

### Description
This PR:
* Modifies component selectors to use kube recommended selectors
* Removes unused constants

### Links
https://issues.redhat.com/browse/LOG-5906

cc @xperimental @Clee2691

### CODE REVIEW

File: test/framework/e2e/fluentd.go
Comment (vparfonov) [inline test/framework/e2e/fluentd.go:1]: can we remove this file?

File: internal/constants/constants.go [hunk 1]
```diff
@@ -117,34 +79,21 @@ const (
 	LabelK8sManagedBy = "app.kubernetes.io/managed-by" // The tool being used to manage the operation of an application (string)
 	LabelK8sCreatedBy = "app.kubernetes.io/created-by" // The controller/user who created this resource (string)
```
Comment (xperimental) [inline internal/constants/constants.go:80]: `LabelK8sCreatedBy` seems to be unused

File: internal/constants/constants.go [hunk 2]
```diff
@@ -117,34 +79,21 @@ const (
...
 
 	HTTPReceiverPort   = 8443
 	HTTPFormat         = "kubeAPIAudit"
 	SyslogReceiverPort = 10514
-
-	LabelHTTPInputService   = "http-input-service"
-	LabelSyslogInputService = "syslog-input-service"
 )
 
 var ReconcileForGlobalProxyList = []string{CollectorTrustedCAName}
```
Comment (xperimental) [inline internal/constants/constants.go:99]: `ReconcileForGlobalProxyList` seems to be unused.

File: internal/constants/constants.go [hunk 3]
```diff
@@ -117,34 +79,21 @@ const (
...
-
-	MigrateInputPrefix         = "migrateInput"
-	MigrateInputApplication    = "migrateInputapplication"
-	MigrateInputInfrastructure = "migrateInputinfrastructure"
-	MigrateInputAudit          = "migrateInputaudit"
-	MigrateDefaultOutput       = "default-replaced"
-	ClusterLoggingAvailable    = "isClusterLoggingAvailable"
+	EventReasonCreateObject = "CreateObject"
+	EventReasonGetObject    = "GetObject"
+	EventReasonRemoveObject = "RemoveObject"
```
Comment (xperimental) [inline internal/constants/constants.go:89]: Seems to be unused:

```suggestion
```

**Review by xperimental — COMMENTED** [review]
Just a few small comments, seems to work fine. Tested with both a "normal CLF" and the LFME.

**Pull request merged**